.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Description
+
+Describe the additions/improvements this PR contains.
+
+# Actions
+
+Provide overview of actions taken to address the additions/improvements.
+
+# Checklist
+
+- [ ] Verified code additions
+- [ ] Updated NatSpec comments (if applicable)
+- [ ] Regenerated docs
```

Comment:
```suggestion
- [ ] Regenerated docs
- [ ] Ran prettier
```

Reply:
Not sure how easy it is to check if docs where regenerated and prettier ran, but I think both should be there.

Reply:
Yeah, this is just for the author of the PR, to check whether these actions were taken or not